packages/@ember/-internals/runtime/lib/system/core_object.js

```diff
@@ -331,12 +331,6 @@ class CoreObject {
     // alerts 'Name is Steve'.
     ```
 
-    NOTE: If you do override `init` for a framework class like `Ember.View`,
-    be sure to call `this._super(...arguments)` in your
-    `init` declaration! If you don't, Ember may not have an opportunity to
-    do important setup work, and you'll see strange behavior in your
-    application.
```

Comment:
This bit is useful in general (and still true). Should we change it to just remove mention of `Ember.View`?

Reply:
Hm, don't we have linting and warnings at runtime for this?

Reply:
Updated :)